modules/standard/List.chpl

```diff
@@ -57,6 +57,13 @@ record list {
   var length: int;
 
   /*
+     Synonym for length.
+   */
+  proc size {
+    return length;
+  }
```

Comment:
I wanted code to be able to use an array or a list without any changes, other than the data type it is working with.